Assets/Scripts/LevelSelect/Physics/PhysicsObject.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DN.Levelselect.PhysicsObjects
+{
+    /// <summary>
+    /// This script can be applied on an object to make it handle the physiscs collision with the car so it can move around or be bumped into with a fun effect.
+    /// </summary>
+    public class PhysicsObject : MonoBehaviour
```

Comment:
Wordt dit script gebruikt? Wat doet `Hit(velocity)`? Is dat een built in functie?

Reply:
nog niet maar dit kan je implementeren op oobjecten om ze een feeling te geven dus wnr je art hebt kan je daar leuke dingen mee doen eventueel

Reply:
en die functie word geroepen in collision script en de parameter is de velocity van de auto